docs-next/pages/apis/fields.mdx

```diff
@@ -510,6 +513,43 @@ export default config({
   /* ... */
 });
 ```
+## File types
+
+File types allow you to upload and store different types of files to your Keystone system.
+
+### file
+
+(coming soon)
+
+### image
+
+An `image` field represents an image file, i.e. `.jpg`, `.png`, `.webp`, or `.gif`.
+
+See [`config.images`](./config#images) for details on how to configure your Keystone system with support for the `image` field type.
```

Comment:
is this going to restrict us from saving images to different target (local/s3/etc) in different list? I see that config.images is keystone wide and I may want to use different target in different list. like avatar in local and gallery in s3 (when available)

Reply:
At the moment the intent is to have a single target location to begin with and give that some time to settle while we observe usage patterns. If there's enough demand we'll investigate expanding the API to support more complex configurations.